cmd/daemon.go

```diff
@@ -99,7 +99,7 @@ func NewCLIDaemon() *CLIDaemon {
 	}
 }
 
-func (cd *CLIDaemon) Start(orchestratorID string, jobID string, clientID string) {
+func (cd *CLIDaemon) Start(orchestratorID string, jobID string, clientID string, isTest ...bool) {
```

Comment:
same comment re: env vars

Reply:
Ah yes this needs redone, I did add CI env variable but didn't finish. Good thanks!